Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM python:3.13-alpine3.21
 
-RUN apk add --no-cache chromium
-RUN pip --no-cache-dir install plotly pandas kaleido==1.0.0rc0 choreographer==0.99.7
+RUN apk add --no-cache chromium git
+RUN pip --no-cache-dir install plotly pandas
+
+ARG kaleido_uri="git+https://github.com/plotly/kaleido@latest-tag#subdirectory=src/py"
+RUN pip install "kaleido>=v1.0.0" || pip install ${kaleido_uri}
```

Comment:
Once Kaleido v1.0.0 is released, this will automatically used that, but for now it uses github